AutonomousAI/data/prompt.txt

```diff
@@ -22,7 +22,7 @@ RESOURCES:
 
 1. Internet access for searches and information gathering
 2. Long Term and Short Term memory management
-3. GPT-4 instances for text generation
+3. GPT-3.5 instances for text generation
```

Comment:
Same comments as before. If the user has access to GPT-4 API, they should be able to use it, yes?

Reply:
Yes